week2/homework/src/index.js

```diff
@@ -1,3 +1,69 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+const fs = require('fs');
+// add command 
+let add = () => {
+  let todo = fs.readFileSync('todos.txt', 'utf8');
+  let text = process.argv[3];
+  fs.appendFileSync('todos.txt', `${text}\n`);
+  return text;
+};
+// list command 
+let list = () => {
+  let todosList = fs.readFileSync('todos.txt', 'utf8');
+  if (!todosList) {
+    console.log('The TODOs list file is empty');
+  } else {
+    console.log(todosList);
+  }
+};
+// remove command
+let remove = () => {
+  let todosList = fs.readFileSync('todos.txt', 'utf8');
+  let todosArray = todosList.split('\n');
+  let commandArg = parseInt(process.argv[3]);
+
+  if (!todosList) {
+    console.log('The TODO list file is empty');
+  } else if (commandArg > todosArray.length) {
+    console.log(`the requested line number ${commandArg} is empty`);
+  } else {
+    todosArray.forEach((element, index) => {
+      if (commandArg === index) {
+        let removedItem = todosArray.splice(index - 1, 1).join('\n');
```

Comment:
```suggestion
        let removedItem = todosArray.splice(index - 1, 1);
```
You splice one item here, so joining one item by new line is not needed.